Svc/FileManager/Commands.fppi

```diff
@@ -36,3 +37,8 @@ async command AppendFile(
                           target: string size 256 @< The name of the file to append to
                         ) \
   opcode 0x05
+
+async command FileSize(
+                        fileName: string size 256 @< The file to get the size of
+                      ) \
```

Comment:
To be consistent, I think we should add an `ignoreErrors` argument here as well, or remove them both and just make them warnings (my preference). Failing to remove a file or get a size is rarely an issue, but returning an error can complicate a long cleanup or listing activity.

Reply:
Should it then be included on the other commands (`CreateDirectory`, `MoveFile`, `RemoveDirectory`, possibly `ShellCommand`, and `AppendFile`) as well, or would it be better to make it a component-wide parameter?